doc/deploy.md

```diff
@@ -91,6 +93,8 @@ You only need to edit the `$DOMAIN` environment variable to point to your public
 The [QuickFeed App installation process](#first-time-installation) will guide you through the rest of the setup,
 setting the environment variables in your `.env` file and saving the `quickfeed.pem` file.
 
+The `PORT` environment variable can be changed to any valid port and could be used to avoid the server from [running on a privileged port](#running-server-on-a-privileged-port). This helpful for when you want to run Quickfeed in debug mode, since you can't input your sudo password into the debug console. A possible workaround to run Quickfeed on privileged port in debug mode is to set capabilities of the  binary you wish to debug with and then start debugging, but keep in mind switching from a privileged port will allow you to run Quickfeed directly without creating a binary - `go run $QUICKFEED/main.go`.
```

Comment:
One sentence per line, please.
There are also language issues (missing words).
I don't think I would frame it as a "possible workaround"; it is an alternative approach.
The last sentence is also too long.
Suggest to ask GPT to check the language and write it more clearly.

Finally, note that there is another section with the flags that are still referring to the now removed `http.addr` flag and a section titled `Running Server on a Privileged Port` that also needs to be updated accordingly.